scripts/create_baseline_stubs.py

```diff
@@ -167,7 +167,7 @@ def main() -> None:
         sys.exit(1)
     project, version = info
 
-    stub_dir = os.path.join("stubs", project)
+    stub_dir = os.path.join("stubs", project).replace("\\", "/")
```

Comment:
Can't we just do this?

```suggestion
    stub_dir = f"stubs/{project}"
```

Reply:
This looks like someone will "refactor"/"clean"/"fix" it to use `os.path.join()` later.

```suggestion
    stub_dir = f"stubs/{project}"  # Must use forward slash in the .json file
```